src/components/EmptyState/EmptyState.scss

```
@@ -162,6 +163,10 @@ $centered-illustration-width: rem(226px);
     text-align: center;
     width: 100%;
   }
+
+  &.fullWidth .Details {
+    max-width: 100%;
+  }
 }
```

We need `fullWidth` to be independent from the `centeredLayout` prop.

```suggestion
}

&.fullWidth {
  .Details {
    max-width: 100%;
  }
}
```